.devcontainer/r-devel-clang-ubsan/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM registry.gitlab.com/jangorecki/dockerfiles/r-devel-clang
+
+RUN apt-get -qq update \
+  && apt-get install -y --no-install-recommends git
+
+# install dependencies without ubsan flags
+RUN Rscript -e 'install.packages(c("bit64", "bit", "R.utils", "xts", "zoo", "yaml", "knitr", "markdown"))'
+
+# setup cc()
+WORKDIR /workspaces/data.table
+
+RUN ln -s /workspaces ~/GitHub
+RUN echo 'Sys.setenv(PROJ_PATH="/workspaces/data.table")\nsource(".dev/cc.R")' >> /workspaces/data.table/.Rprofile
+
+# set ubsan flags
+RUN mkdir -p ~/.R \
+        && echo 'CC=clang -fsanitize=undefined -fno-sanitize=float-divide-by-zero -fno-sanitize=alignment -fno-omit-frame-pointer' > ~/.R/Makevars \
```

Comment:
maybe keep a Makevars file in this directory and use `COPY` to bring it in to the container? That way the Makevars is stored in plain text/with syntax highlighting, don't have to visually parse it from inside the bash commands.

Reply:
Done in current commit